pertpy/tools/_distances/_distances.py

```diff
@@ -203,28 +225,71 @@ def __call__(
 
         return self.metric_fct(X, Y, **kwargs)
 
+    def bootstrap(
+        self,
+        X: np.ndarray,
+        Y: np.ndarray,
+        *,
+        n_bootstrap: int = 100,
+        bootstrap_random_state: int = 0,
```

Comment:
```suggestion
        random_state: int = 0,
```

This should be good enough, right? The function already has the name `bootstrap` and users would expect this not to be passed somewhere else.